plexpy/notifiers.py

```diff
@@ -2721,3 +2748,71 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+class HIPCHAT(object):
+
+    def __init__(self):
+        self.apiurl = plexpy.CONFIG.HIPCHAT_URL
+        self.color = plexpy.CONFIG.HIPCHAT_COLOR
+
+    def notify(self, message, subject):
+        if not message or not subject:
+            return
+        
+        text = '(plex) ' + subject.encode('utf-8') + ': ' + message.encode('utf-8')
```

Comment:
Any of the notifications agents where subject and message are sent in one string should have the option to turn off the subject line. See Telegram/Slack/Twitter/Facebook/etc.

Also, why is `(plex)` added to the beginning?